src/Microsoft.TestPlatform.ObjectModel/AttachmentSet.cs

```diff
@@ -40,6 +40,11 @@ public AttachmentSet(Uri uri, string displayName)
             DisplayName = displayName;
             Attachments = new List<UriDataAttachment>();
         }
+
+        public override string ToString()
+        {
+            return $"{nameof(Uri)}: {Uri.AbsoluteUri}, {nameof(DisplayName)}: {DisplayName}, {nameof(Attachments)}: [{ string.Join(",", Attachments)}]";
```

Comment:
can Uri be null?

Reply:
Not sure about that. However we are using AbsoluteUri in [other places](https://github.com/smadala/vstest/blob/2cfc8a2abc01adb740bd9f89c7cbcffd69e4eaac/src/Microsoft.TestPlatform.Extensions.TrxLogger/Utility/Converter.cs#L164) without any check. Not adding null check here.